src/nspawn/nspawn.c

```diff
@@ -1158,13 +1162,53 @@ static int mount_all(const char *dest, bool userns) {
         return 0;
 }
 
+static int parse_mount_bind_options(const char *options, unsigned long *mount_flags, char **mount_opts) {
+        const char *p = options;
+        unsigned long flags = *mount_flags;
+        char *opts = NULL;
+
+        assert(options);
+
+        for (;;) {
+                _cleanup_free_ char *word = NULL;
+                int r = extract_first_word(&p, &word, ",", EXTRACT_QUOTES);
```

Comment:
@poettering I don't know if this use of `EXTRACT_QUOTES` is correct here. I think we want to honor quotes so options could have commas in it.